package.json

```diff
@@ -26,17 +26,17 @@
     "test": "grunt test"
   },
   "dependencies": {
-    "clone": "~0.1.11",
-    "parserlib": "~0.2.4"
+    "clone": "~0.2.0",
+    "parserlib": "~0.2.5"
   },
   "devDependencies": {
-    "grunt": "~0.4.4",
-    "grunt-contrib-clean": "~0.5.0",
-    "grunt-contrib-concat": "~0.4.0",
-    "grunt-contrib-copy": "~0.5.0",
-    "grunt-contrib-jshint": "~0.10.0",
+    "grunt": "~0.4.5",
+    "grunt-contrib-clean": "~0.6.0",
+    "grunt-contrib-concat": "~0.5.0",
+    "grunt-contrib-copy": "~0.7.0",
+    "grunt-contrib-jshint": "~0.11.0",
     "grunt-contrib-watch": "~0.6.1",
-    "grunt-include-replace": "~1.2.0",
+    "grunt-include-replace": "~2.0.0",
```

Comment:
When it's ready to be merged I will update it then.
On Feb 28, 2015 9:54 PM, "Shinnosuke Watanabe" <notifications@github.com>
wrote:

> In package.json
> <https://github.com/CSSLint/csslint/pull/512#discussion_r25560623>:
>
> >      "grunt-contrib-watch": "~0.6.1",
> > -    "grunt-include-replace": "~1.2.0",
> > +    "grunt-include-replace": "~2.0.0",
>
> 3.0.0 now https://www.npmjs.com/package/grunt-include-replace
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CSSLint/csslint/pull/512/files#r25560623>.
>